--- a/src/hotspot/os/linux/os_linux.cpp
+++ b/src/hotspot/os/linux/os_linux.cpp
@@ -138,7 +138,7 @@
 // and has been around since glibc 2.0.
 extern "C" {
-static void dlvsym(void* handle,
+static void* dlvsym_workaround(void* handle,
                    const char* name,
                    const char* version) {
   return ::dlvsym(handle, name, version);
@@ -3081,7 +3081,7 @@
 
   void *ret = NULL;
   if (version != NULL) {
-    ret = dlvsym(handle, name, version);
+    ret = dlvsym_workaround(handle, name, version);
   } else {
     ret = dlsym(handle, name);
   }
